en/docs/tutorials/create-your-first-rest-api.md

```diff
@@ -52,78 +56,81 @@ The `Statistics` REST API opens on a separate page.
+3. Add a new resource by clicking the **+** icon under the existing construct.
 
-        Select HTTP as the service type, and enter the following information.
+   ![Add resource](../assets/img/tutorials/rest-api/add-resource.png){.cInlineImage-half}
 
-        | **Field**             | **Value**             |
-        |-----------------------|-----------------------|
-        | **Service Base Path** | `COVID19-Statistics`  |
-        | **Listener Port**     | `9090`                |
+    Then click **Resource**. 
 
-        Select the **Define Inline** check box, and then click **Save**.
+    As a result, a panel named **Configure Resource** appears on the right of the page. In this panel, enter information as follows:
 
-    2. Expand the construct you created by clicking on it. Then click the **EDIT** icon of the default `GET` resource within the construct.
+     | **Field**       | **Value**                |
+     |-----------------|--------------------------|
+     | **HTTP Method** | `GET`                    |
+     | **Path**        | `stats/[string country]` |
+     | **Return Type** | `json|error?`            |
 
-    3. Enter `stats/[string country]` in the **Path** field.
+    Here, you are configuring a GET resource to fetch statistics related to COVID-19.
 
-        By adding `string country` within square brackets in the path, you are introducing `country` as a path parameter for which the value should be in the string format.
+    By adding `string country` within square brackets in the path, you are introducing `country` as a path parameter for which the value should be in the string format.
 
-    4. Click **Advanced** and in the **Return Type** field, enter `json|error` to get the return in `json` or `error1` format.
+    The specified return type allows the REST API to return the output in either JSON or error? format.
 
-    5. Click **Save API**.
+    Click **Save**.
 
-3. To connect to the COVID-19 API and retrieve data, follow the sub-steps below.
+    The low-code diagram of this resource opens .
 
-    1. Click on the API resource you edited, and then click on the first **+** icon after the **Start** statement.
+4. To connect to the COVID-19 API and retrieve data, follow the sub-steps below.
 
-    2. Click **Connector** and then select **Covid19 API**.
+    1. Click **Connector** to add a connector. Then search for the COVID-19 connector, and click on it once it appears in the search results.
 
-    3. In the **Covid19 API Connection** window, enter `covid19Client` as the **Connection Name** and click **Continue to Invoke API**.
+    2. In the **Connector** panel, `covid19Client` as the **Connection Name** and click **Continue to Invoke API**.
 
-    4. In the **Operation** drop-down list, select **getStatusByCountry** and enter details as follows in the other fields:
+       3. In the **Covid19 API Connection** window, enter `covid19Client` as the **Connection Name** and click **Continue to Invoke API**.
```

Comment:
Numbering alignment doesn't seem right.

Also if the name of the window includes Covid19, shall we report an issue to get it corrected to COVID-19.

Reply:
Corrected the numbering. Shall we leave the window name in documentation unchanged for now  until  it is corrected in the UI? Will create an issue for that.